codice/alliance
# CAL-215 Adds security audit plugin

#### What does this PR do?
Adds a security audit plugin with a configuration for security attributes that should be audited when changed.

Looking for feedback on the audit message, specifically, whether or not it should say something else and in addition if there seems there is anything missing from the configuration file. 

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@codice/security 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@lessarderic 
@stustison

#### How should this be tested?
Ingest anything into the catalog and then change a security attribute (this is easier with the Catalog UI in DDF). Verify the change was audited. 

#### What are the relevant tickets?
[CAL-215](https://codice.atlassian.net/browse/CAL-215)
#### Checklist:
